test_scripts/API/SetAppIcon/006_Two_apps_are_registered_App1_iconResumed_true_App2_iconResumed_true.lua

```diff
@@ -0,0 +1,74 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0041-appicon-resumption.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) SDL, HMI are started.
+-- 2) App1 set custom icon via putfile and SetAppIcon requests.
+-- 3) App2 set custom icon via putfile and SetAppIcon requests.
+-- 4) Two app are re-registered.
+-- SDL does:
+-- 1) Register an App 1 successfully, respond to RAI with result code "SUCCESS", "iconResumed" = true.
+-- 2) Register an App 2 successfully, respond to RAI with result code "SUCCESS", "iconResumed" = true.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SetAppIcon/commonIconResumed')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  syncFileName = "icon.png"
```

Comment:
@HSavynetska Propose to set 2 different icons for the Apps in order to make sure that each app will be resumed with it's own icon

Reply:
Done